src/helpers.ts

```diff
@@ -2606,7 +2606,14 @@ data: ${data}
     } = getFrom(options.from);
 
     let tx;
-    const deployment = await partialExtension.get(name);
+    let deployment = await partialExtension.getOrNull(name);
+    if (!deployment) {
+      if (!options.to) throw new Error(`expected deployment for ${name} or option "to" with deploymed address`)
+      deployment= {
+        address: options.to,
+        abi: (await partialExtension.getArtifact(name)).abi
```

Comment:
The issue here is that name is used for artifact.
I think a more explicit method would be better

Reply:
Yeah, but at the same time the 'to' parameter, derived from 'TxOptions', is not used though it's available to put it.
Thus, the PR puts a value and meaning in 'to' option.
Idk if another method should exist for this as this is not a breaking change and only extends the use of this function.
What do you think?